strawberry/flask/tests/schema.py

```diff
@@ -0,0 +1,17 @@
+import strawberry
+
+
+@strawberry.type
+class User:
+    name: str
+    age: int
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def user(self, info) -> User:
+        return User(name="Patrick", age=100)
+
+
+schema = strawberry.Schema(query=Query)
```

Comment:
I think we have this somewhere else already, can we make it into a fixture (maybe it already is) and reuse it? :)

Reply:
should be in the tests for django I think